collections/_posts/2024-11-13-AlphaFold2.md

```diff
@@ -0,0 +1,208 @@
+<p style="text-align: center;font-style:italic">Figure 11. Training procedure of AlphaFold.</p>
+
+&nbsp;
+
+# Main ablation study
+
+The authors provide an ablation study of the main components and design choices of AlphaFold
+
+<div style="text-align:center">
+<img src="/collections/images/AlphaFold/ablation_study.jpg" width=800></div>
+<p style="text-align: center;font-style:italic">Figure 12. Ablation study of some of the components of AlphaFold.</p>
+
+&nbsp;
+
+# Main results
+
+The most important evaluation of AlphaFold abilities was done during the CASP14 challenge. The competiting methods were evaluated on never-seen-before proteins and associated 3D structure, to ensure that their generalization capabilities. The metrics used for the evaluation, such as the GDT score and IDDT-C$$\alpha$$, are not discussed here, but the key takeaway message is the AlphaFold achieves a **sub atomic** precision, with a median r.m.s.d of 0.95 $$A$$ (Angstrom) for the backbone atoms and of 1.5 $$A$$ for all the atoms. 
+
+<div style="text-align:center">
+<img src="/collections/images/AlphaFold/results_casp14.jpg" width=300></div>
+<p style="text-align: center;font-style:italic">Figure 13. Median all-atom RSMD on the CASP14 set of protein domains (N = 87 protein domains) relative to the top-15 entries.</p>
+
+
+We showcase hereinbelow some examples of predictions of 3D structure of moolecules by AlphaFold with their ground truth experimentally obtained spatial arrangement (in grey, or blue for the first two images). These images are actually not only proteins but also RNAs : they are predictions from the next iteration of Google Deepmind's model, AlphaFold3, which can predict the spatial structure of a broader variety of biomolecules.
+
+<div style="text-align:center">
+<img src="/collections/images/AlphaFold/ex1.jpg" width=600></div>
+
+<div style="text-align:center">
+<img src="/collections/images/AlphaFold/ex2.jpg" width=600></div>
+
+<div style="text-align:center">
+<img src="/collections/images/AlphaFold/ex3.jpg" width=600></div>
+
+<div style="text-align:center">
+<img src="/collections/images/AlphaFold/ex4.jpg" width=600></div>
+
+<div style="text-align:center">
+<img src="/collections/images/AlphaFold/ex5.jpg" width=600></div>
+
+&nbsp;
+
+# What's next ?
+
+Google Deepmind did not take part in any of the challenges that followed CASP14, but since the models at the top of ladder are all modified versions of AlphaFold2. As already briefly introduced before, Deepmind has recently released, in collaboration with Isomorphic Labs, [AlphaFold3](https://www.nature.com/articles/s41586-024-07487-w), which can provide the spatial structure of a wider variety of biomolecules and with an improved accuracy compared to AlphaFold2. They have created a database with the structure of 200 millions of biomolecules predicted by their model, and allow any researcher to upload their own sequence to get the prediction of AlphaFold3. 
```

Comment:
```suggestion
Google Deepmind did not take part in any of the challenges that followed CASP14, but since then the best models have all been modified versions of AlphaFold2. As mentioned before, Deepmind has recently released, in collaboration with Isomorphic Labs, [AlphaFold3](https://www.nature.com/articles/s41586-024-07487-w), which can provide the spatial structure of a wider variety of biomolecules and with an improved accuracy compared to AlphaFold2. They have created a database with the structures of 200 millions of biomolecules predicted by their model, and allow any researcher to upload their own sequence to get a prediction from AlphaFold3. 
```